local-cluster/src/local_cluster.rs

```diff
@@ -972,6 +982,10 @@ impl LocalCluster {
 
         Ok(tpu_client)
     }
+
+    fn required_validator_funding(stake: u64) -> u64 {
+        stake.saturating_mul(2).saturating_add(2)
```

Comment:
this number may also change because the validator itself will need fees to pay for votes, etc.

Reply:
Nope, all this local-cluster seems like a whole bunch of hacked together things.
Trying to keep this PR as a pure refactor without changing any values